image-index.md

```diff
@@ -112,5 +117,7 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 }
 ```
 
+[cpufeatures.h]: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/x86/include/asm/cpufeatures.h
```

Comment:
Fixed “os.features” → “features” in dc5e4a0 → e1045bf.  Also replaced the old `sse4` instances, although I replaced them with `fpu` since I was using `fpu` and `vme` since those are the first two entries in `cpufeatures.h`.  I don't really care what features we use as examples though; if there's a strong preference for `sse4_2`, I can switch to that instead.